src/components/button.component.js

```diff
@@ -0,0 +1,218 @@
+import React, { Component } from 'react';
+import { Platform } from 'react-native';
+import { Button as BaseButton } from 'react-native-elements';
+import { colors, fonts, normalize } from 'config';
+
+const types = {
+  default: {
+    buttonStyle: {
+      borderColor: '#adadad',
+      borderWidth: 1,
```

Comment:
This property can be removed everywhere, because in `defaultButtonStyle` it is defined?

Reply:
oupsy :D never interrupt a refactoring.
thanks for catching this